### ISSUE TITLE
[Feature request] Add more calendar events

### ISSUE THREAD

**Issue by tomduud** [danielbrendel/hortusfox-web#160]
I can't find a way to edit calendar event topics. I think I need at least "other", "harvest" and "replace" events. It would be great if these can be added. There might be other events but these came to my mind.

**Comment by danielbrendel** [comment]
Thanks for the hint, we can add these, too! 

What do you mean with "replace"? Maybe we can choose a better wording.

**Comment by tomduud** [comment]
Certainly, there will be a better word for it. I mean with "replace" to replace filter medium or similar consumable item. Or use it with plants to replace them with new ones after harvest. For salads/microgreen etc there is no need to add them as a "plant" since their lifespan is limited.

**Comment by danielbrendel** [comment]
I am tempted to use the term "respawn" for that. :D I am not a native english speaker, but in german it does not only translate to "resurrect" but also "restart" which would essentially describe the process.

**Comment by danielbrendel** [comment]
You can now manage custom calendar classes. I consider this specific topic resolved.
